docs/Configuration.md

```diff
@@ -108,7 +108,9 @@ Group memberships can be set on user entry or group entry or both.
 
 ## Configuration of ACEs
 
-The configurations are done per principal followed by indented settings for each ACE. This data includes
+The configurations are done per principal (given through its name) followed by indented settings for each ACE. Currently only principal names are supported which are equal to one of the created group ids.
```

Comment:
@kwin this is not entirely true - while usually principalName=authorizableId when using the AC Tool, for the case the externalId (think LDAP) is used, the section `ace_config` has to list the authorizableId (while the tool sets automatically and correctly the corresponding principalName to the content paths)